app/Http/Controllers/ProfilesController.php

```diff
@@ -342,5 +344,20 @@ public function restore(Profile $profile)
         $profile->restore();
 
         return redirect()->route('profiles.table')->with('flash_message', 'The profile of ' . $profile->full_name . ' has been restored.');
-    } 
+    }
+
+    /**
+     * Generate PFD Export
+     *
+     * @param  Profile $profile
+     * @return pdf
+     */
+    public function pdfExport(Profile $profile) {
+       $pdf_content =  Browsershot::url($profile->url)
+            ->setIncludePath("/Users/bxc180019/.nvm/versions/node/v16.15.1/bin")
```

Comment:
I had to include my local path to the node.js modules that Browsershot needs to create the pdf file. Still need to figure out whether is necessary to specify this path on the server as well.

Reply:
Discussion #80